AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md

```diff
@@ -153,11 +153,10 @@ Create a new kernel for the Jupyter notebook based on your activated conda envir
 > Linux*:
 Go to the section which corresponds to the installation method chosen in [AI Tools Selector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-tools-selector.html) to see relevant instructions:
 * [AI Tools Offline Installer (Validated)](#ai-tools-offline-installer-validated)
-* [Conda/PIP](#condapip) 
-* [Docker](#docker)
+* [Conda/PIP](#setup-conda-environment)
 
-### AI Tools Offline Installer (Validated)  
-1. If you have not already done so, activate the AI Tools bundle base environment. 
+### AI Tools Offline Installer (Validated)
+1. If you have not already done so, activate the AI Tools bundle base environment.
```

Comment:
Suggested content for AI Tools Offline Installer section:
 
**1. Register Conda kernel to Jupyter Notebook kernel**

If the default path is used during the installation of AI Tools:
```
$HOME/intel/oneapi/intelpython/envs/tensorflow-gpu/bin/python -m ipykernel install --user --name=tensorflow-gpu
```
If a non-default path is used:
```
<custom_path>/bin/python -m ipykernel install --user --name=tensorflow-gpu
```
**2. Launch Jupyter Notebook** 
<!-- add other flags to jupyter notebook command if needed, such as port 8888 or allow-root -->
```
jupyter notebook --ip=0.0.0.0
```
**3. Follow the instructions to open the URL with the token in your browser**

**4. Select the Notebook**
<!-- add sample file name -->
```
inc_sample_tensorflow.ipynb
```
**5. Change the kernel to `tensorflow-gpu`**
  <!-- specify relevant kernel name(s), for example `pytorch` -->
**6. Run every cell in the Notebook in sequence**

Reply:
yes, update